src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -9076,10 +9050,8 @@ void VRFTerminalUnitEquipment::ControlVRFToLoad(EnergyPlusData &state,
     int VRFCond = this->VRFSysNum;
     Real64 FullOutput = 0.0;   // unit full output when compressor is operating [W]
     Real64 TempOutput = 0.0;   // unit output when iteration limit exceeded [W]
-    int SolFla = 0;            // Flag of RegulaFalsi solver
     Real64 TempMinPLR = 0.0;   // min PLR used in Regula Falsi call
     Real64 TempMaxPLR = 0.0;   // max PLR used in Regula Falsi call
-    bool ContinueIter;         // used when convergence is an issue
```

Comment:
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:9079]:(style),[variableScope],The scope of the variable 'SolFla' can be reduced.
[src/EnergyPlus/HVACVariableRefrigerantFlow.cc:9082]:(style),[variableScope],The scope of the variable 'ContinueIter' can be reduced.